src/main/java/DukeComponent/Ui.java

```diff
@@ -0,0 +1,58 @@
+package DukeComponent;
```

Comment:
Mentioned earlier but you can consider moving all your printline stuff here since this class is meant to interact with the user.